harness/determined/keras/_tf_keras_trial.py

```diff
@@ -486,6 +486,7 @@ def run(self) -> None:
             if wkld.kind == workload.Workload.Kind.RUN_STEP:
                 # Store the train_response_func for later.
                 self.train_response_func = response_func
+                self.batches_per_step = wkld.batches_per_step
```

Comment:
Can we make this `self.num_batches` as well?

Reply:
Totally, makes a lot more sense.